src/physics/em/detail/SBPositronXsCorrector.hh

```diff
@@ -24,14 +24,16 @@ namespace detail
  * section by a factor
  * \f[
    \frac{\sigma_\mathrm{corrected}}{\sigma}
-       = \exp( \alpha Z [ \beta(E) - \beta(E_c) ] )
- \f]
- * where \f$ \beta \f$ is the Lorentz factor: \f[
-  \beta(E) = \frac{E + m_e c^2}{\sqrt{E * E + 2 * E * m_e * c^2}} \,,
+       = \exp( \alpha Z [ \beta^{-1}(K_i - E') - \beta^{-1}(K_i - E_c) ] )
  * \f]
- * \f$ \alpha \f$ is the fine structure constant, \f$E_c\f$ is the gamma
- * production cutoff energy, and \f$ E \f$ is the provisionally sampled exiting
- * energy.
+ * where the inverse positron speed is : \f[
+  \beta^{-1}(K) = \frac{c}{v} = \sqrt{1 - \left( \frac{m_e c^2}{K + m_e c^2} \right)^2}
+ \,,
+ * \f]
+ * \f$ \alpha \f$ is the fine structure constant, \f$K_i\f$ is the
+ * incident positron energy, \f$E_c\f$ is the gamma
+ * production cutoff energy, and \f$ E' \f$ is the provisionally sampled exiting
+ * energy of the positron.
```

Comment:
```suggestion
 * \f$ \alpha \f$ is the fine structure constant, \f$K_i\f$ is the
 * incident positron energy, \f$E_c\f$ is the gamma
 * production cutoff energy, and \f$ E' \f$ is the provisionally sampled exiting
 * energy of the photon.
```
It might also be helpful to use the same notation as in the rest of the bremsstrahlung documentation, i.e. `E` for the incident charged particle energy and `k` for the emitted photon energy.

Reply:
OK, I was trying to be careful about "K" for kinetic energy vs "E" for rest + kinetic -- since the latter is used in the beta calculation. Take a look at the update.

Reply:
Makes sense.. but that `E'` in the first inverse beta term should be the sampled energy of the photon, not positron, right?